Raven.Voron/Voron/Trees/Page.cs

```diff
@@ -172,6 +172,19 @@ public ushort PageSize
             }
         }
 
+        public List<long> GetAllOverflowPages()
+        {
+            var results = new List<long>(NumberOfEntries);
```

Comment:
Ahhhh now I understand what you mean... I can ask the NodeHeader itself located on this page I am working on if it is an overflow or not... and filter accordingly. Am I right?